Makefile

```diff
@@ -16,40 +18,62 @@ CGO_ENABLED=0
 ##############################
 .PHONY: cluster/prepare
```

Comment:
@slaskawi @pb82 I'm still going through the code and testing it. But I just wonder if it would be more interesting if we moved part of the logic to some shell scripts.

I know that the Openshift team does something similar https://github.com/openshift/origin/blob/master/Makefile#L57 and you can see all the scripts here: https://github.com/openshift/origin/tree/master/hack.

I'm just thinking that as the project grows, the Makefile may become cluttered with several alternatives.

Does it make any sense?

Reply:
I agree, we can do that.